### ISSUE TITLE
Extended support for Android "Safe Area" (area around the notch)

### ISSUE THREAD

**Issue by NickIliev** [NativeScript/NativeScript#7724]
**Is your feature request related to a problem? Please describe.**
The newer Android devices (e.g., Pixel 3, Samsung S10 Galaxy, Huawei P20, P30) are now coming with a notch and an area similar to iOS's safe area. Provide means for controlling the content in the area around the notch.

**Comment by fpaaske** [comment]
I guess this is even more relevant now, when Android starts enforcing edge-to-egde feature when targeting SDK 35: https://developer.android.com/develop/ui/views/layout/edge-to-edge

> Once you target SDK 35 or higher on a device running Android 15 or higher, your app is displayed edge-to-edge. The window spans the entire width and height of the display by drawing behind the system bars. System bars include the status bar, caption bar, and navigation bar.

**Comment by asharghi** [comment]
I think @triniwiz is working on a solution?

**Comment by fpaaske** [comment]
Yes, we had a quick chat on discord earlier today. He's da man! 👍

**Comment by djmaze** [comment]
Any solution for this?

**Comment by djmaze** [comment]
Ah,  there is a WIP PR: https://github.com/NativeScript/NativeScript/pull/10774
